Blanks_eol.py

```diff
@@ -0,0 +1,23 @@
+import os
+import re
+
+def trim_right_spaces(text):
+    return text.rstrip()
+
+def main():
```

Comment:
You need to add a docstring for every function that you create. You may follow [this](https://github.com/apache/hadoop/blob/9c8cdbe28e317e703560c3eefcde2f6eebcabb17/dev-support/docker/pkg-resolver/check_platform.py#L28-L30).